janusgraph-examples/example-cassandra/pom.xml

```diff
@@ -0,0 +1,37 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.janusgraph</groupId>
+        <artifactId>janusgraph-examples</artifactId>
+        <version>0.2.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>example-cassandra</artifactId>
+    <packaging>pom</packaging>
+    <name>Example-Cassandra: C* Thrift Storage, ES Index</name>
```

Comment:
`/ES/Elasticsearch/`

Reply:
I can do this (expand C* also), but I was trying to keep the formatting of the Maven build completion message to fit nicely ;)